Makefile

```diff
@@ -51,13 +51,13 @@ run:
 # dep targets - dependency management
 
 dep-install:
-	@echo Installing dep itself
+	@echo Installing Glide itself
 	@mkdir -p ${GOPATH}/bin
-	@hack/get-go-dep.sh
+	@curl https://glide.sh/get | sh
```

Comment:
Rather than running this script, could we not just download the glide binary directly and put it in the $GOPATH/bin? That way we can specify the version and md5sum.

Reply:
We could. I just did it like this to ensure we pull down the latest release. But if we want to pin it to a certain release, we can write our own glide.sh-like script and pull what we want (the real glide.sh doesn't allow you to specify a version - it always pulls latest, as defined by https://glide.sh/version ).